lib/jekyll-relative-links/generator.rb

```diff
@@ -21,16 +21,52 @@ def initialize(site)
     end
 
     def generate(site)
+      # Get any configuration specified
+      # Defaults mean no change in behaviour
+      jrl_config = site.config["jekyll_relative_links"] || {}
+      @config = {}
+      @config["collections"] = jrl_config["collections"] || []
+      @config["process_all_collections"] = jrl_config["process_all_collections"] || false
+      @config["verbose"] = jrl_config["verbose"] || 0
+
       @site    = site
       @context = context
 
       site.pages.each do |page|
         next unless markdown_extension?(page.extname)
         replace_relative_links!(page)
       end
+
+      site.collections.each do |collection|
+        label = collection[1].label
+        if @config["process_all_collections"] || @config["collections"].include?(label)
+          update_collection(label)
+        elsif @config["verbose"] > 0
+          puts "Skipping #{label} because process_all_collections is false"\
+               " and collection name isn't in config"
+        end
+      end
+    end
+
+    def update_collection(collection_name)
+      # Make sure that the metadata for this collection says that we're creating output
+      if site.collections[collection_name].metadata["output"]
+        if @config["verbose"] > 0
+          puts "Processing #{collection_name}"
+        end
+        site.collections[collection_name].docs.each do |page|
+          next unless markdown_extension?(page.extname)
+          replace_relative_links!(page)
+        end
+      elsif @config["verbose"] > 0
+        puts "Skipping #{collection_name} because output is false"
+      end
     end
 
     def replace_relative_links!(page)
+      if @config["verbose"] > 1
+        puts "Replacing links in #{page.path}"
```

Comment:
Verbose logging is theoretically out of scope for this PR, but since it's small, I'm fine including it... if we do, however, we should use `Jekyll.logger.debug` and default to the Jekyll-wide log level.